packages/cozy-procedures/src/templates/creditApplicationTemplate.js

```diff
@@ -131,31 +131,114 @@ const documents = {
     label: 'identity_document',
     order: 1,
     count: 1,
-    rules: {}
+    rules: {
+      $or: [
+        {
+          mime: 'text/plain'
+        },
+        {
+          mime: 'application/pdf'
+        },
+        {
+          class: 'image'
+        }
+      ],
+      metadata: {
+        classification: 'identity_document'
+      }
+    }
   },
   tax_notice: {
     label: 'tax_notice',
     order: 2,
     count: 1,
-    rules: {}
+    rules: {
+      $or: [
+        {
+          mime: 'text/plain'
+        },
+        {
+          mime: 'application/pdf'
+        }
+      ],
+      metadata: {
+        classification: 'tax_notice',
+        subject: 'income'
+      }
+    }
   },
   payslip: {
     label: 'payslip',
     order: 3,
     count: 3,
-    rules: {}
+    rules: {
+      $or: [
+        {
+          mime: 'text/plain'
+        },
+        {
+          mime: 'application/pdf'
+        }
+      ],
+      metadata: {
+        classification: 'payslip'
+      }
+    }
   },
   bank_identity: {
     label: 'bank_identity',
     order: 4,
     count: 1,
-    rules: {}
+    rules: {
+      $or: [
+        {
+          mime: 'text/plain'
+        },
+        {
+          mime: 'application/pdf'
+        }
+      ],
+      metadata: {}
+    }
   },
   address_certificate: {
     label: 'address_certificate',
     order: 5,
     count: 1,
-    rules: {}
+    rules: {
+      $or: [
+        {
+          mime: 'text/plain'
+        },
+        {
+          mime: 'application/pdf'
+        }
+      ],
+      metadata: {
+        $and: [
```

Comment:
Why do we need the $and operator here? There's only one item in the array, right?

Reply:
You right